Dockerfile

```diff
@@ -0,0 +1,456 @@
+# syntax=docker/dockerfile:1.3
+# NB: at least 1.3 is needed to benefit from ARG expansion in bind mount arguments
+ARG PYTHON_VERSION=3.9.16
+
+# Driver args
+# print on gpu is broken for 22.43.24595.30 + igc-1.0.12812.26
+ARG CR_TAG=22.43.24595.30
+ARG IGC_TAG=igc-1.0.12504.5
+ARG CM_TAG=latest
+ARG L0_TAG=latest
+
+# ONEAPI
+ARG ONEAPI_INSTALLER_URL=https://registrationcenter-download.intel.com/akdlm/irc_nas/19079
```

Comment:
Is this something we have keep changing with every oneAPI release?

Reply:
I guess so. I'd like to find way how to deal with it, just setting ONEAPI_VERSION, not build number as well